packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -76,26 +78,34 @@ export default defineConfig({
       const mintedL1Erc20Token = await l1Erc20Token.mint()
       await mintedL1Erc20Token.wait()
 
-      // Send minted ERC-20 to the test wallet
+      // Send minted ERC-20 to the test localWallet
```

Comment:
I think that should say `Send minted ERC-20 to the test userWallet`?

Reply:
Done